functions/bouncer/src/middleware/errorHandler.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import * as Sentry from '@sentry/node';
+
+export default function errorHandler(err, req, res, next) {
+  const { whRes } = req;
+  whRes.stages.push('errorHandler');
+  Sentry.setContext('whRes', JSON.stringify(whRes));
+
+  // for unknown errors
+  if (err instanceof Error) {
+    console.log(`errorHandler unknown error: ${err.message}`);
```

Comment:
dlog?

Reply:
This is gcloud function and we explicitly want this message (and others like it) to end up in the functions logs for troubleshooting assistance.